### PR TITLE
:arrow_up: [Kotlin] Update against Kotlin 1.4

### PR DESCRIPTION
**Pull request by kittinunf**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (bohsen) [inline gradle/wrapper/gradle-wrapper.properties:3]: [Gradle 6.6](https://gradle.org/releases/) was released a week ago, if you're interested in the latest stable release.

File: gradle.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 org.gradle.parallel=true
 
-kotlinVersion=1.3.72
+kotlinVersion=1.4.0
 
 bintray=1.8.4
 coroutines=1.3.3
```
[the remark this answers is not in the document]
Comment (kittinunf) [reply gradle.properties:7]: Thanks for the heads-up! 👍

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (kittinunf) [reply gradle/wrapper/gradle-wrapper.properties]: Fixed! 👏

File: gradle.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (kittinunf) [reply gradle.properties]: Done

**Pull request merged**